app/controllers/admin/petitions_controller.rb

```diff
@@ -10,7 +10,7 @@ def show
   end
 
   def threshold
-    @petitions = Petition.threshold.order(:signature_count).paginate(:page => params[:page], :per_page => params[:per_page] || 20)
+    @petitions = Petition.threshold.order(:signature_count).paginate(:page => params[:page], :per_page => 50)
```

Comment:
What Otto said, although this method will disappear once my branch https://github.com/alphagov/e-petitions/pull/212 gets merged in (removing threshold and todolist).